### ISSUE TITLE
FileName with spaces are tricky to handle in makefile

### ISSUE THREAD

**Issue by Kit-Law-42** [se2p/ps-wise-2021-program-description#21]
Hi. I have a question regarding the makefile included in the assignment package.
Are we allowed to slightly modify it to better handle some exceptions? Such as handling spaces in the input file.
And to ensure that the makefile trigger a command e.g. "node minesweeper.js "file   Name123.cfg" without throwing exceptions in the command prompt.

**Comment by alessiogambi** [comment]
Hi, no you are not allowed to modify the makefile.
Or better, you can modify it locally, but do not commit it to the repo. Otherwise, it might break the automated testing.

Can you elaborate a bit more on the issue? Maybe, if this is something relevant for the upcoming assignments, I can include it there.

**Comment by alessiogambi** [comment]
@alexlaw19 I close this issue for the moment, in case you need it, please re-open it

**Comment by Kit-Law-42** [comment]
> @alexlaw19 I close this issue for the moment, in case you need it, please re-open it

Sorry for replying late. 
In my case, I got the failure when I tested the minesweeper.js when with File named"File   Name.cfg" as the arguments.
and parser of the command prompt can only handle this with extra quotes"" on commands.
e.g. node minesweeper.js "File   Name.cfg" (Good)
node minesweeper.js File   Name.cfg (Exception is thrown)

Can we assume that we would not have a filename with spaces in the test case?
Or should I treat them as 2 inputs to our js file?
